test/integration/scheduler_perf/util.go

```diff
@@ -107,15 +122,30 @@ func mustSetupScheduler(ctx context.Context, b *testing.B, config *config.KubeSc
 
 	// Not all config options will be effective but only those mostly related with scheduler performance will
 	// be applied to start a scheduler, most of them are defined in `scheduler.schedulerOptions`.
-	_, podInformer := util.StartScheduler(ctx, client, cfg, config)
+	_, informerFactory := util.StartScheduler(ctx, client, cfg, config)
 	util.StartFakePVController(ctx, client)
 
-	shutdownFn := func() {
-		cancel()
-		tearDownFn()
+	// Testing of DRA with inline resource claims depends on these
+	// controllers for creating and removing ResourceClaims.
+	if withDynamicResourceAllocation {
```

Comment:
I put this here because of `util.StartFakePVController`. That is another additional controller run in the background by `mustSetupScheduler`.

A separate helper function for the code makes sense. I don't care where that then gets called, but it should be consistent with calling `util.StartFakePVController`. Do you prefer both in `mustSetupScheduler` or both in `runWorkload`?